ui-particles-angular/projects/ui-particles-angular/src/scss/gio-mat-palettes.scss

```diff
@@ -156,6 +230,7 @@ $mat-decorative-palette: mat.define-palette(
 );
 
 // Method color
+// @Deprecated
```

Comment:
Are we sure about this ? It's useful to have same POST GET badge with same colour 🤷‍♂️